boilerplates/fe-react/README.md

```diff
@@ -2,79 +2,30 @@
 
 React quickstarter project
 
- ## Purpose of this quickstarter
-
- This quickstarter generates a React project for developing webpages.
-
- It contains the basic setup for [Docker](https://www.docker.com/), [Jenkins](https://jenkins.io/), [SonarQube](https://www.sonarqube.org/) and [OpenShift](https://www.openshift.com/), so you have your CI/CD process out of the box.
-
-
- ## What files / architecture is generated?
-
- ```
-├── Jenkinsfile - This file contains Jenkins build configuration settings
```

Comment:
if the file structure changed, update, but don't move the files structure

Reply:
I added a direct link to create-react-app file structure